wombat-admin/src/components/incoming_requests.tsx

```diff
@@ -34,74 +25,55 @@ export default function IncomingRequests({
               <table className="min-w-full divide-y divide-zinc-300">
                 <thead>
                   <tr>
-                    <th
-                      scope="col"
-                      className="py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-zinc-800 sm:pl-0"
-                    >
-                      ID
-                    </th>
-                    <th
-                      scope="col"
-                      className="px-3 py-3.5 text-left text-sm font-semibold text-zinc-800"
-                    >
-                      Name
-                    </th>
-                    <th
-                      scope="col"
-                      className="px-3 py-3.5 text-left text-sm font-semibold text-zinc-800"
-                    >
-                      Role
-                    </th>
-                    <th
-                      scope="col"
-                      className="px-3 py-3.5 text-left text-sm font-semibold text-zinc-800"
-                    >
-                      Check In
-                    </th>
-                    <th
-                      scope="col"
-                      className="px-3 py-3.5 text-left text-sm font-semibold text-zinc-800"
-                    >
-                      Image
-                    </th>
+                    <th className="py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-zinc-800 sm:pl-0">ID</th>
```

Comment:
Nit: Formatting fight :( We should put a prettier config in place to avoid these refactors.

Reply:
haven't fixed yet. will work on it on the next SCRUM

Reply:
looks good, i'd say run `npx prettier --write .` then push that up